recipes-demos/ti-apps-launcher/ti-apps-launcher.bb

```diff
@@ -98,6 +98,12 @@ do_install:append() {
         install -d ${D}${systemd_system_unitdir}
         install -m 0755 ${WORKDIR}/ti-demo.service ${D}${systemd_system_unitdir}/ti-demo.service
     fi
+
+    if [ "${SERVICE_SUFFIX}" == "-eglfs" ]; then
+        install -d ${D}/etc/udev/rules.d/
```

Comment:
@sskartheekadivi  Better to use "Yocto Project Openembedded directories installation variables" - `${D}${sysconfdir}/udev/rules.d/` [1]

[1]: https://wiki.koansoftware.com/index.php/Directories_and_installation_variables